### PR TITLE
Migrate snekclientv2 aa2 to implement 1

### PR DESCRIPTION
**Pull request by pinterid**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the front does not work with the latest engine.

**What is the new behavior (if this is a feature change)?**
- Now the front works with the latest engine.

**Other information**:
- Client v2: https://github.com/snek-at/client/releases/tag/0.2.0-rc.5

### CODE REVIEW

**Review by Aichnerc — APPROVED** [review]
LGTM
I really think the syntax for `throw Error()` should be changed to `yeet Error()` in the next major JS release.

File: src/store/actions/userActions.js [hunk 1]
```diff
@@ -198,22 +198,24 @@ const readCacheAction = (username) => {
                 };
               }
 
+              console.log(data)
+
```
Comment (pinterid) [inline src/store/actions/userActions.js]: Remove lines.
```suggestion
```

File: src/store/actions/userActions.js [hunk 2]
```diff
@@ -233,10 +235,12 @@ const readCacheAction = (username) => {
             }
           }
         })
-        .catch((ex) =>
+        .catch((ex) => {
+          console.log(ex)
```
Comment (pinterid) [inline src/store/actions/userActions.js]: Remove line.
```suggestion
```

File: src/store/actions/userActions.js [hunk 3]
```diff
@@ -233,10 +235,12 @@ const readCacheAction = (username) => {
...
           dispatch({ type: "READ_CACHE_ERROR", payload: { error: ex } })
-        );
+        });
     } catch (ex) {
+      console.log(ex)
```
Comment (pinterid) [inline src/store/actions/userActions.js]: Remove line.
```suggestion
```

**Review by schettn — APPROVED** [review]
lgtm

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**